test/manual/data_lake.test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const { expect } = require('chai');
+const path = require('path');
+const { TestRunnerContext } = require('../functional/spec-runner');
+const { gatherTestSuites } = require('../functional/spec-runner');
+const { generateTopologyTests } = require('../functional/spec-runner');
+const { withClient } = require('../functional/shared');
+
+describe('Atlas Data Lake', function () {
+  context('spec tests', function () {
+    const testContext = new TestRunnerContext({
+      skipInit: true,
+      useSessions: false,
+      user: 'mhuser',
+      password: 'pencil'
+    });
+
+    let testSuites = gatherTestSuites(path.resolve(__dirname, '../spec/atlas-data-lake-testing'));
+
+    after(() => testContext.teardown());
+    before(function () {
+      return testContext.setup(this.configuration);
+    });
+
+    generateTopologyTests(testSuites, testContext);
+  });
+
+  describe('prose Tests', function () {
+    it('should properly constructs and issues a killCursors command', function () {
+      return withClient('mongodb://mhuser:pencil@localhost', (client, done) => {
```

Comment:
Does the withClient have to be called? like `withClient(...)()` or will it be since its returned? Just want to make sure the expectations are being run.

Reply:
Darn, got me, thanks for the catch. I want to re-write this `withClient` thing so badly, ugg.